File: Examples/test-suite/csharp/director_min_runme.cs
using System;

namespace director_minNamespace {

public class runme {
  static void Main() {
    MinCallback callback = new MinCallback();
    Caller caller = new Caller(callback);
    if (!caller.call())
       throw new Exception("Should return true");
  }
}

class MinCallback : Callback
{
  public override bool run()
  {
    return true;
  }
}

}

File: Examples/test-suite/ruby/director_min_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#

require 'swig_assert'

require 'director_min'

class MinCallback < Director_min::Callback
  def run()
    return true
  end
end

callback = MinCallback.new()
caller = Director_min::Caller.new(callback)
raise RuntimeError, "Should return true" unless caller.call()
